src/views/Supply.ts

```diff
@@ -103,4 +103,29 @@ export namespace Supply {
         type: PrintType;
         quantity: number;
     }
+
+    export const typeName = (type: string) => {
```

Comment:
We should create a translate function for each type.

Reply:
I will implement i18 when the page will be released

Reply:
We are still lacking the table in the "i can help" flow, so there is some time left